ci/scripts/support-rota-notification/index.js

```diff
@@ -30,6 +30,14 @@ async function updateSupportTopics () {
   }
 }
 
+function isABankHoliday (now) {
+  //todo: use https://www.gov.uk/bank-holidays.json to get bank holiday dates
+  const bankHolidays = ['25-12-2024', '26-12-2024', '1-Jan-2025', '18-04-2025', '05-05-2025', '26-05-2025',
```

Comment:
```suggestion
  const bankHolidays = ['25-12-2024', '26-12-2024', '01-01-2025', '18-04-2025', '05-05-2025', '26-05-2025',
```

Not sure this will work with "-Jan-"

Reply:
Good spot! udpated